versacommerce_app.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.description = %q{This generator creates a basic sessions controller for authenticating with your Shop. Also a home controller which lets you access your shop easily.}
 
   s.add_runtime_dependency('rails', '>= 3.1')
```

Comment:
@kozubversacommerce We should set this to `'>= 5.1'`

Reply:
fixed